### PR TITLE
Endpoint for getting all crops

### PR DESCRIPTION
**Pull request by zenkaeser**

### CODE REVIEW

**Comment by MirSzabo** [comment]
tested in postman with GET method and works as expected, good work!

File: src/server/config/swagger.json [hunk 1]
```diff
@@ -1,11 +1,47 @@
...
+  "paths": {
+    "/crops": {
+      "get": {
+        "tags": [
+          "crops"
+        ],
+        "summary": "/api/crops",
+        "description": "Get all crops",
+        "operationId": "getAllCrops",
+        "produces": [
```
Comment (cecastosic) [inline src/server/config/swagger.json:25]: You already said that API can produce application/json, maybe here you can define example response, for example {
  "application/json": {
       "id": "2",
			"name": "Shiso ",
			"plant_variety": "Lamiaceae",
			"fk_user_id": "2",
			"created_at": "2020-03-16 21:03:56",
			"updated_at": "2020-03-16 21:03:56",
			"deleted_at": null
}

File: src/server/api/controllers/crops.controller.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const knex = require('../../config/db');
+
+const getAllCrops = async () => {
+  try {
+    return await knex('crops').select('crops.name', 'crops.plant_variety');
```
Comment (bytewiz) [inline src/server/api/controllers/crops.controller.js:5]: FUN FACT: you don't need to do both return AND await. It should resolve to the same thing 🚀

**Comment by senner008** [comment]
@bytewiz should we merge this?

**Comment by senner007** [comment]
@zenkaeser 
Can you resolve these conflicts?

**Comment by ricardoaguiar** [comment]
I'm going to add the swagger-auto docs to this endpoint.

**Comment by dpfernandes** [comment]
Approved

**Pull request merged**